packages/react-page-renderer/src/renderer/AbstractClientPageRenderer.ts

```diff
@@ -222,6 +228,50 @@ export default abstract class AbstractClientPageRenderer extends AbstractPageRen
     );
   }
 
+  /**
+   * Batch patch promise values to controller state.
+   */
+  private _startBatchTransactions(
+    controller: ControllerDecorator,
+    patchedPromises: { [key: string]: Promise<unknown> }
```

Comment:
Zkontroluj si prosim te ty typy...nektere jsou definovane univerzalne (UnknownPromiseParameters)

Reply:
Ten typ je trošku jinej než UnknownPromiseParameters ne? To bych pak měl přepstat i v mount (pageResources),  _patchPromisesToState (patchedPromises), atd.. nebo jsem to jen nepochopil?